src/XR/features/WebXRControllerPhysics.ts

```diff
@@ -87,46 +87,55 @@ export class WebXRControllerPhysics extends WebXRAbstractFeature {
         // if no motion controller available, create impostors!
         if (this._options.physicsProperties!.useControllerMesh && xrController.inputSource.gamepad) {
             xrController.onMotionControllerInitObservable.addOnce((motionController) => {
-                motionController.onModelLoadedObservable.addOnce(() => {
-                    const impostor = new PhysicsImpostor(motionController.rootMesh!, PhysicsImpostor.MeshImpostor, {
-                        mass: 0,
-                        ...this._options.physicsProperties,
+                if (!motionController._doNotLoadControllerMesh) {
+                    motionController.onModelLoadedObservable.addOnce(() => {
+                        const impostor = new PhysicsImpostor(motionController.rootMesh!, PhysicsImpostor.MeshImpostor, {

```

Comment:
nit: You could put each parameter on its own line instead of having a long line here

Reply:
Oh hah, my word wrap did that visually for me so I didn't even notice.